tests/logging/test_terse_json.py

```diff
@@ -78,7 +78,7 @@ def test_log_output(self):
             "server_name",
             "name",
         ]
-        self.assertEqual(set(log.keys()), set(expected_log_keys))
```

Comment:
Why this change?

Reply:
While debugging the code this test was failing, but using `assertCountEquals` gives a better error message. 🤷 I can back it out if you want.

Reply:
Ah right. It's just the new test is weaker, so I'd prefer we kept it as it is if possible.

Reply:
`assertCountEqual` should be a *stronger* test:

> Test that sequence *first* contains the same elements as *second*, regardless of their order. When they don’t, an error message listing the differences between the sequences will be generated.
>
> Duplicate elements are *not* ignored when comparing *first* and *second*. It verifies whether each element has the same count in both sequences. Equivalent to: `assertEqual(Counter(list(first)), Counter(list(second)))` but works with sequences of unhashable objects as well.

Casting to a `set` first means that duplicates are allowed.

How do you see it being a weaker test?

Reply:
Oooooooooooooh, sorry, for some reason I was sure it was just testing the lengths matched. Ignore me